docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -2,488 +2,716 @@
+     user@intkey-tp$ sudo -u sawtooth intkey-tp-python -v tcp://127.0.0.1:4004
 
-Run the following commands:
+* The IntegerKey and XO transaction processors are available in several
+  languages in addition to Python. For more information, see
+  :doc:`../transaction_family_specifications`.
 
-.. code-block:: console
 
-  $ intkey create_batch
-  Writing to batches.intkey...
+Step 6: Confirm that the REST API is Running
+============================================
 
-  $ intkey load
-  batches: 2 batch/sec: 135.96900883377907
+In order to configure the validator, the REST API must be running.
 
-You can watch the processing of the transactions by observing the
-logging output of the ``intkey`` transaction processor. A truncated example of
-this output is shown below:
+.. The client terminal window is used to run Sawtooth commands, which interact
+.. with the validator through the REST API.
 
-.. code-block:: console
+#. Open a new terminal window (the client terminal window). In this
+   procedure, the prompt ``user@client$`` shows the commands that should be
+   run in this window.
 
-  [19:29:26 INFO    core] register attempt: OK
-  [19:31:06 INFO    handler] processing: Verb=set Name=eBuPof Value=99811 address=1cf126c584128aaf1837c90c83748ab222c11b8bbd2fe6cc30f17fe35f2acb9af8efd4ee3f092b676546316cf85b2e929b68d9c5314e93ac318ba527ec74aa3ed1bc2e
-  [19:31:06 INFO    handler] processing: Verb=set Name=HOUUQS Value=10140 address=1cf126380fa9e716a05ac815741fd1960d5952e60f8747e13334f79504c57d0287b77cf9b78284d0e1544f6f0366d66c6e6eb99dc5c154b84175b2d20008d721c7b623
-  [19:31:06 INFO    handler] processing: Verb=set Name=lrnuDC Value=92318 address=1cf12617c797cf8c27254bbdb5c9bda09f9405b9494ae32b79b9b6d30881ca8552d5932a68f703d1b6754b9feb2edafa76a797fc0826110381b0f8614f2c6853316b47
-  [19:31:06 INFO    handler] processing: Verb=set Name=BKaiql Value=94175 address=1cf12669cbc17d076a1accb4b0bb61f40ed4f999173b90e3ca2591875a55fee2947661e60fa1c57b41ef0f2660176b945a01c85ff645543297068a3fb1306324a19612
-  [19:31:06 INFO    handler] processing: Verb=set Name=wpMQmE Value=47316 address=1cf1260f6bdf66b65ff7c00ec58c4deccffd167bfee7a85698880dfa485df3de1ec18a5b2d1dc12849743d1c74320108360a2d40d223b35fbc1c4ea03bbd8306480c62
-  [19:31:06 INFO    handler] processing: Verb=set Name=GTgrvP Value=31921 address=1cf12606ac7db03c756133c07d7d02b59f3ef9eae6774fe59c75c88ab66a9fabbbaef9975dbf9aa197d1090ed126d7b18e2
+#. Run the following command:
 
+   .. code-block:: console
 
-Viewing Blocks and State
-========================
+      user@client$ ps aux | grep [s]awtooth-rest-api
+      sawtooth  2829  0.0  0.3  55756  3980 pts/0    S+   19:36   0:00 sudo -u sawtooth sawtooth-rest-api -v
+      sawtooth  2830  0.0  3.6 221164 37520 pts/0    Sl+  19:36   0:00 /usr/bin/python3 /usr/bin/sawtooth-rest-api -v
+
+#. If necessary, run the following command to restart the REST API.
+
+   .. code-block:: console
+
+      user@client$ sudo -u sawtooth sawtooth-rest-api -v
```

Comment:
I would link back to the step for starting the REST API instead of repeating content from that section.